hkmc2DiffTests/src/test/scala/hkmc2/MLsDiffMaker.scala

```diff
@@ -64,7 +65,7 @@ abstract class MLsDiffMaker extends DiffMaker:
   val stackSafe = Command("stackSafe")(_.trim)
   val liftDefns = NullaryCommand("lift")
   val importQQ = NullaryCommand("qq")
-  val staging = NullaryCommand("staging")
+  val stageCode = NullaryCommand("ds")
```

Comment:
Why is the command named "ds"? I would expect this shorthand to mean "debug staging".